### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mrsteve0924**
added dependency list for ArchLinux

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,15 @@ This contains an install script called `install.sh`. It is a script to install a
 
 ## Dependencies
 
-The following is a list of dependencies needed on Ubuntu, similar lists are required on other distributions. The last one is only needed if you want to install WCM.
+Dependency List for Ubuntu. Similar lists are required on other distributions. Package hwdata is only needed if you want to install WCM.
```
Comment (soreau) [inline README.md:6]: hwdata is not a requirement for wcm, as far as I am aware. It looks like it's actually a [dependency of libdisplay-info](https://gitlab.freedesktop.org/emersion/libdisplay-info/-/blob/main/meson.build?ref_type=heads#L19).

Comment (soreau) [reply README.md:6]: Trailing whitespace should be fixed and I don't think we use vala, either. wcm only depends on [wayfire and wf-shell](https://github.com/WayfireWM/wcm/blob/master/meson.build#L6-L7).

Comment (mrsteve0924) [reply README.md:6]: the current list indicated that hwdata is needed to wcm, but i can take that reference out. 
valac is in the current list which is provided by vala.  i can take it out from the original list if needed. 
 
i just took the current list of ubuntu deps and crossed them to arch packages

Comment (soreau) [reply README.md:6]: > the current list indicated that hwdata is needed to wcm, but i can take that reference out. valac is in the current list which is provided by vala. i can take it out from the original list if needed.
> 
> i just took the current list of ubuntu deps and crossed them to arch packages

We might as well clean this up and strive for correctness while you're modifying it. You might compare this list with what's found in the [CI runners](https://github.com/WayfireWM/wayfire/blob/master/.github/workflows/ci.yaml) too.

**Pull request closed without merging**